lib/CHeroHandler.h

```diff
@@ -121,6 +122,7 @@ class DLL_LINKAGE CHero
 		h & iconSpecLarge;
 		h & portraitSmall;
 		h & portraitLarge;
+		h & battleImage;

```

Comment:
Must be new format version (like in lines below)

Reply:
Lines below ? 
"std::vector<std::shared_ptr<Bonus>> SpecialtyInfoToBonuses(const SSpecialtyInfo & spec, int sid);"

That format ?

Reply:
No

Reply:
```
if(version >= 759)
{
	h & identifier;
}
```

Reply:
Also change SERIALIZATION_VERSION in CSerializer.h